examples/nlp/cai/utils.py

```diff
@@ -0,0 +1,89 @@
+import os
+from collections import defaultdict
+from multiprocessing import Pool
+
+from tqdm import tqdm
```

Comment:
NeMo-Aligner is run from within the NeMo container, and 'tqdm' is present in the NeMo container's [requirements.txt](https://github.com/NVIDIA/NeMo/blob/8e65042d15062ce3fbe639f9d428c639510d894c/requirements/requirements.txt#L13)

Reply:
This is more of a nit, since I highly doubt `tdqm` will ever be removed. I just wanted to point out that it's better practice to declare the packages you explicitly import and let `pip` decide if a re-install is needed. You could simply leave it unpinned in https://github.com/NVIDIA/NeMo-Aligner/blob/main/requirements.txt. What you're guarding against is if upstream repos don't use that package anymore, then your code is broken. It also makes explicit what your project expects

Reply:
While I do agree, I would like to again stress that, as with the container, if this is something that we would like to address then this is not specific to this algorithm/tutorial, and we might want to fix it framework-wide. It is not a new requirement. Thus, I'm wondering if there was some other consideration not to add it so far. 

e.g. `tqdm` is also being used by other algorithms within NeMo-Aligner and wasn't added to the `requirements.txt`. 
https://github.com/NVIDIA/NeMo-Aligner/blob/main/examples/nlp/data/steerlm/attribute_annotate.py#L36

Reply:
Since I classify this as a nit, let me just resolve this discussion as "fix later"; this shouldn't be a reason not to merge this.